src/thunderbots/software/ai/passing/pass.cpp

```diff
@@ -0,0 +1,55 @@
+/**
+ * Implementation of the "Pass" class
+ */
+
+#include "pass.h"
+
+using namespace AI::Passing;
+
+// Pass::Pass()  :
```

Comment:
Unless this commented-out code is going to be used we should remove it.

Reply:
changed